### PR TITLE
2023.01.01 website

### PR DESCRIPTION
**Pull request by richard-thonig**

### CODE REVIEW

**Comment by richard-thonig** [comment]
@timtroendle all seems to be well. Hurray.
Note: I included a (not too pretty) new page Explore data, that references to cspdata.com

File: csp-guru/datapackage.json [hunk 1]
```diff
@@ -32,6 +32,11 @@
                         "description": "Country where the CSP plant is located",
                         "type": "string"
                     },
+                    {
+                        "name": "Region",
+                        "description": "Worldregion where the CSP plant is located",
```
Comment (timtroendle) [inline csp-guru/datapackage.json:37]: World region

File: csp-guru/datapackage.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
...is data if you give appropriate credit, under the Creative Commons Attribution 4.0 International (CC BY 4.0) license.\n\nFor academic and professional use (presentations, journal articles, reports, etc):if possible, please cite the source in which the present dataset was first used [Lilliestam et al. (2021)](  https://doi.org/10.1080/15567249.2020.1773580). Please also link to [www.csp.guru](https://www.csp.guru) if possible.\n\n## Contact\n\nFor questions or issues get in touch with [Richard Thonig](mailto:richard.thonig@iass-potsdam.de).\n## Acknowledgements\nFunding for this version  came from the SolarPACES secretariat. Previous versions of csp.guru were funded by the European Research Council Consolidator Grant [DESERTECTION (grant number 313553)](https://cordis.europa.eu/project/rcn/106709_de.html) for version 2016-08-30; csp.guru versions 2018-05-14 and csp.guru version 2019-09-01 received support from the European Union Horizon 2020 research and innovation program [MUSTEC (grant agreement No 764626) ](https://cordis.europa.eu/project/rcn/211264_en.html).\n\n## Archive\n\nAll versions of the CSP.guru dataset are archived at [Zenodo](https://doi.org/10.5281/zenodo.1318151).",
```
Comment (timtroendle) [inline csp-guru/datapackage.json:7]: `1. January` -> `1 January`

**Review by timtroendle — APPROVED** [review]
I only looked at the changes to the metadata and website and they look good to me.

**Comment by JaviBonilla** [comment]
> [JaviBonilla](https://github.com/JaviBonilla) check out the netlify preview https://deploy-preview-107--csp-guru.netlify.app/explore.html - this is how I would link to cspdata.com should we include anything else?

@richard-thonig, it looks good to me.

I prepared another image with a transparent background logo.

banner

**Pull request merged**